web/index.html

```diff
@@ -21,7 +21,7 @@
     <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="packages/devtools/debugger/debugger.css">
     <link rel="stylesheet" href="packages/devtools/logging/logging.css">
-
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Note that all the rest of our deps are in-lined into the package - that they could be served w/o an internet connection (including the octicons icons). Perhaps open an issue to in-line the material icons font?

Reply:
I can't seem to make a comment down there, but right after `<!-- add the known pages -->`, can you add a place for the Inspector page, and remove the one for the Device page? We have an approximation of the final UI in static html so that there's less flashing and re-layout when the app launches.

Reply:
Good catch. I'd thought that part of static HTML was cruft to cleanup rather than server side rendering./

Reply:
Done. Loaded the flutter icons locally.